packages/arb-bridge-eth/test/arbRollup.js

```diff
@@ -216,18 +221,26 @@ class Assertion {
   }
 
   deadline() {
-    return 13000 * this.blockNumber + grace_period_ticks;
+    return 1000 * this.blockNumber + grace_period_ticks;
```

Comment:
I'm not sure what this is for, but it's not the formula that the EthBridge uses to compute challenge deadlines.

Reply:
I changed it to be closer to correct and added a comment. Maybe at some point it'll be worth doing more, but the only use case of this is getting transactions accepted in the ethbridge for gas cost analysis which this achieves.